src/Uno.UI.RemoteControl.Host/Program.cs

```diff
@@ -158,36 +161,53 @@ static async Task Main(string[] args)
 				globalServiceProvider.GetService<IIdeChannel>();
 
 #pragma warning disable ASPDEPR004
-				// WebHostBuilder is deprecated in .NET 10 RC1.
-				// As we still build for .NET 9, ignore this warning until $(NetPrevious)=net10.
-				// https://github.com/aspnet/Announcements/issues/526
+				// STEP 2: Create the WebApplication builder with reference to the global service provider
+				var builder = WebApplication.CreateBuilder(new WebApplicationOptions
+				{
+					Args = args,
+					ContentRootPath = Directory.GetCurrentDirectory(),
+				});
 
-				// STEP 2: Create the WebHost with reference to the global service provider
-				var builder = new WebHostBuilder()
-					.UseSetting("UseIISIntegration", false.ToString())
+				// Configure Kestrel and URLs
+				builder.WebHost
 					.UseKestrel()
 					.UseUrls($"http://*:{httpPort}/")
 					.UseContentRoot(Directory.GetCurrentDirectory())
-					.UseStartup<Startup>()
-					.ConfigureLogging(logging => logging
-						.ClearProviders()
-						.AddConsole()
-						.SetMinimumLevel(LogLevel.Debug))
+					.ConfigureLogging(logging =>
+					{
+						logging.ClearProviders();
+						logging.AddConsole();
+						logging.SetMinimumLevel(LogLevel.Debug);
+						logging.AddFilter("Microsoft.", LogLevel.Information);
+						logging.AddFilter("Microsoft.AspNetCore.Routing", LogLevel.Warning);
+					})
 					.ConfigureAppConfiguration((hostingContext, config) =>
 					{
 						config.AddCommandLine(args);
+						config.AddEnvironmentVariables("UNO_PLATFORM_DEVSERVER_");
```

Comment:
Really nice, love the idea, however usually we prefix our environment variables with only `UNO_`

```suggestion
						config.AddEnvironmentVariables("UNO_DEVSERVER_");
```